src/components/Consul/ConsulStartNewButton.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Button } from '../Button';
+import { Wrapper } from '../Wrapper';
+import { ScreenName } from '../../types';
+
+export const ConsulStartNewButton = ({ title, buttonTitle, query, navigation, data }) => {
```

Comment:
do we really need this new component instead of using `<Wrapper><Button ... /></Wrapper>` elsewhere? with having it exported/imported. what if we do not need the wrapper at some place but on the other? it would be more difficult now? what do you think?

Reply:
I updated the code. I just didn't want to wrap it with Wrapper inside every file I use. I used it directly in the component.

Reply:
[fix(consul): navigation error](https://github.com/ikuseiGmbH/smart-village-app-app/pull/499/commits/75ec5e17ed952a2123a3f26b7b3c308270f7d934)

Reply:
but now what is the difference to `Button`? you could use `Button` everywhere you use `ConsulStartNewButton` now i guess?

Reply:
Yes that's true. I wanted to use it as a component because I used it in several places. I just updated the code and removed this component.